docs/deployment/config_descriptions.md

```diff
@@ -4,51 +4,106 @@ title: Configuration Descriptions
 hide_title: true
 ---
 
-# IAM Configuration
+# Cumulus Configuration
 
-## iam-deployment-name
+## Overview
 
-The name (e.g. dev) of the the 'deployment' - this key tells kes which configuration set (in addition to the default values) to use when creating the cloud formation template[^1]
+The table below provides an overview of the `config.yml` variables.
+Note that entries delimited as \<name\> are intended to be read as objects where `name` is the key, not the value, e.g.:
 
-## prefix
-
-This value will prefix CloudFormation-created IAM resources and permissions. **The `stackName` used in the [app](deployment/deployment-readme#configure-and-deploy-the-cumulus-stack) deployment must start with this prefix or the deployment will not work correctly.**
-
-## stackName
-
-The name of this iam stack in CloudFormation (e.g. <prefix>-iam).
-
-**The cumulus stack name must start with `<prefix>`** [^2]
-
-## buckets
-
-The buckets created in the [Create S3 Buckets](#create-s3-buckets) step. Buckets are defined in the config.yml with a key, name, and type. Types should be one of: internal, public, private, or protected. Multiple buckets of each type can be configured. A key is used for the buckets to allow for swapping out the bucket names easily.
+```yaml
+# Config for 'dynamos.\<name\>.read' where `name = UsersTable`
+dynamos:
+  UsersTable:
+    read: 5
+```
 
-## useNgapPermissionBoundary
+### config.yml Explained
+
+| field | default     | description
+| ----- | ----------- | -----------
+| prefix | (required) | the name used as a prefix in all aws resources
+| prefixNoDash | (required) | prefix with no dash
+| users | | List of URS usernames permitted to access the Cumulus dashboard/API
+| urs_url | `https://uat.urs.earthdata.nasa.gov/` | URS url used for OAuth
+| useNgapPermissionBoundary | false | Required to be `true` when deploying to the NGAP platform
+| useWorkflowLambdaVersions | true | Version deployed lambdas when they are updated.
+| cmr.username | (required) | the username used for posting metadata to CMR
+| cmr.provider | CUMULUS | the provider used for posting metadata to CMR
+| cmr.clientId | CUMULUS | the clientId used to authenticate with the CMR
+| cmr.password | (required) | the password used to authenticate with the CMR
+| buckets | (required) | Configuration of buckets with key, bucket name, and type (i.e. internal, public private)
+| system_bucket | `buckets.internal.name` | the bucket used for storing deployment artifacts
+| shared_data_bucket | cumulus-data-shared | bucket containing shared data artifacts e.g. Cumulus message adapter
+| ems.provider | CUMULUS | the provider used for sending reports to EMS
+| vpc.vpcId | (required if ecs is used) | the vpcId used with the deployment
```

Comment:
`vpc` settings are also required now if you are using NGAP, right? That maybe doesn't need to be spelled out here, but maybe we should create a follow-up ticket to validate that or something

Reply:
I would expect that is generally true and generally known. We can handle it whichever way you prefer.

Reply:
Fair enough. We probably don't need to cover every contingency in this doc